srcts/src/shiny/index.ts

```diff
@@ -606,6 +606,24 @@ class ShinyClass {
       sendOutputHiddenState
     );
 
+    function doSendWindowSize() {
+      inputs.setInput(".clientdata_window_width", window.innerWidth);
+      inputs.setInput(".clientdata_window_height", window.innerHeight);
+      inputs.setInput(
+        ".clientdata_scroll_width",
+        document.documentElement.scrollWidth
+      );
+      inputs.setInput(
+        ".clientdata_scroll_height",
+        document.documentElement.scrollHeight
+      );
+    }
+
+    sendImageSizeFns.setImageSend(inputBatchSender, doSendWindowSize);
+    sendImageSizeFns.regular(); // Initial send
```

Comment:
I don't think we should be immediately invoking `.regular()` (or `doSendWindowSize()`) here, but instead, we should be adding to `initialValues` similar to how the other initial client data values are populated below.